content/chat/replies.textile

```diff
@@ -0,0 +1,122 @@
+---
+title: Replies and Threads
+meta_description: "Implement the ability to reply to or create a thread based on any previously sent messages in your chat application."
+product: chat
+languages:
+  - javascript
+---
+
+Reply to a message directly with a reference to the original message, or create threads on a message to separate the new branch of messages from the main path of messages. Replies enable members to quickly and directly respond to an existing message. Threads enable members to create a thread for a specifically focused conversation between two or more members within the channel.
+
+h2(#enable). Enable replies and threads
+
+To use replies and threads, you'll need to enable this feature as a new "channel rule":/channels#rules within your application in the dashboard.
+
+Namespaces enable your chat to be grouped based on a prefix included as part of the channel name. Defining namespaces helps manage channels, such as defining the speed at which members can post messages or whether members can interact with messages such as reactions, replies, or threads.
```

Comment:
This is about namespaces but adds little value to how the description of how to do message replies and threads. It probably should be mentioned since it helps with configs but does it need 2 paragraphs?